cadc-vos/src/main/java/ca/nrc/cadc/vos/Transfer.java

```diff
@@ -91,76 +88,46 @@
      */
     public int version = VOS.VOSPACE_20;
 
-    private URI target;
+    public List<URI> targets = new ArrayList<URI>();
+    public List<Protocol> protocols = new ArrayList<Protocol>();
+
     private Direction direction;
     private View view;
-    private Long contentLength;
-    private String remoteIP;
-    private List<Protocol> protocols;
     private boolean keepBytes = true;
     private boolean quickTransfer = false;
-
-    protected Transfer() { }
-
-    /**
-     * Constructor for copying a node.
-     * @param target
-     * @param direction
-     * @param protocols
-     */
-    public Transfer(URI target, Direction direction, List<Protocol> protocols)
-    {
-        this.target = target;
-        this.direction = direction;
-        this.protocols = protocols;
-        if (this.protocols == null)
-            this.protocols = new ArrayList<Protocol>();
-    }
+    private Long contentLength;
+    private String remoteIP;
 
     /**
      * Constructor for internal vospace move and copy operations.
      *
-     * @param target
-     * @param destination
+     * @param target - URI of target of transfer
+     * @param destination - URI of destination of transfer
      * @param keepBytes true for copy, false for move
      */
-    public Transfer(URI target, URI destination, boolean keepBytes)
-    {
-        this.target = target;
+    public Transfer(URI target, URI destination, boolean keepBytes) {
+        this.targets.add(target);
         this.direction = new Direction(destination.toASCIIString());
         this.keepBytes = keepBytes;
-        this.protocols = new ArrayList<Protocol>();
     }
 
     /**
-     * Constructor for copying a node with a specific view.
-     *
-     * @param target
-     * @param direction
-     * @param view
-     * @param protocols
+     * Ctor for Transfer. Use this for setting up transfers using multiple targets.
+     * Caller must call getProtocols().add(...) and getTargets().add(...)
```

Comment:
Yes but it's no longer getProtocols() and getTargets(), it's protocols.add() and targets.add()